lib/compiler.js

```diff
@@ -65,6 +65,13 @@ exports.compile = function (code, options) {
     });
 
     importExportVisitor.finalizeHoisting();
+
+    if (getOption(options, "wrapWithRun")) {
+      result.code = "module.run(function(){" +
+        magicString.toString() + "//*/\n});";
+
+      return result;
+    }
```

Comment:
Without `wrapWithRun` things break so doesn't seem like much of an option. Let's just use `module.run` and roll with it.